spec/models/test_track/analytics/safe_wrapper_spec.rb

```diff
@@ -5,34 +5,18 @@
 
   subject { TestTrack::Analytics::SafeWrapper.new(underlying) }
 
-  describe '#alias' do
```

Comment:
Change this to a spec for `#sign_up!` along with the noop behavior if its not implemented?